modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java

```diff
@@ -710,6 +710,90 @@ private void _createFDSSortObjectDefinition(
 			ObjectRelationshipConstants.TYPE_ONE_TO_MANY, null);
 	}
 
+	private ObjectDefinition _createFDSViewObjectDefinition(
+			Locale locale, long userId)
+		throws Exception {
+
+		ObjectDefinition fdsViewObjectDefinition =
+			_objectDefinitionLocalService.addSystemObjectDefinition(
+				"FDSView", userId, 0, "FDSView", null, false, true,
+				LocalizedMapUtil.getLocalizedMap("FDS View"), true, "FDSView",
+				null, null, null, null,
+				LocalizedMapUtil.getLocalizedMap("FDS Views"), false,
```

Comment:
We are reusing `FDSView` as entity that now holds `Data Set`s. The main reason for this is to avoid large amount of refactoring. **This is temporary.** As a part of [changing all entity names](https://liferay.atlassian.net/browse/LPD-6838) we will rename this to something new, like `DSMDataSet`.

Reply:
Makes sense, persistence should focus around that, it's the cheapest option.

Renaming persistence is a different business